src/vivarium_data_requests/jun_17_laura_lamberti/README.md

```diff
@@ -0,0 +1,20 @@
+A follow-up request from Laura Lamberti of BMGF on behalf of Hao Hu.
```

Comment:
We should do the directories as YYYY_MM_DD_client.  It'll keep the data requests sorted and make sure we don't have overlap if laura happens to request something else a year from now.